### PR TITLE
Add KYC document archived webhook

### PR DESCRIPTION
**Pull request by subenson**
Add the KYC document archive functionality to the general description

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -104,6 +104,7 @@ tags:
       If it is not, or if you the `credit-file-proof` isn't requested, then redirect your customer to the KYC document gatherer to collect their KYC documents.
       Rebilly parses and analyzes the documents and accepts or rejects them according to a configurable scoring algorithm.
       When all of the document types in the KYC request are accepted, the status is considered fulfilled and the KYC request fulfilled webhook will be emitted.
+      If a document is accepted while unreviewed documents of the same type exist, these unreviewed documents will be archived and the archived webook will be emitted.
```
Comment (subenson) [inline openapi/openapi.yaml:107]:
```suggestion
      If a document is accepted while unreviewed documents of the same type exist, the unreviewed documents will be archived and the archived webook will be emitted.
```

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -104,6 +104,7 @@ tags:
       If it is not, or if you the `credit-file-proof` isn't requested, then redirect your customer to the KYC document gatherer to collect their KYC documents.
       Rebilly parses and analyzes the documents and accepts or rejects them according to a configurable scoring algorithm.
       When all of the document types in the KYC request are accepted, the status is considered fulfilled and the KYC request fulfilled webhook will be emitted.
+      If a document is accepted while unreviewed documents of the same type exist, the unreviewed documents will be archived and the archived webook will be emitted.
```
Comment (akozhevnikov-rebilly) [inline openapi/openapi.yaml:107]:
```suggestion
      If a document is accepted while unreviewed documents of the same type exist, the unreviewed documents will be archived and the archived webhook will be emitted.
```

Comment (noon-dawg) [inline openapi/openapi.yaml:107]: I'm not sure what's meant by "emitted"? I think this word is ambiguous. 
I think we should use "removed" or "executed".

Comment (ashkarpetin) [reply openapi/openapi.yaml:107]: More likely "sent".

Comment (noon-dawg) [reply openapi/openapi.yaml:107]: I'm not sure if "sent" is the best word to use relating to webhooks. I suppose it depends on if we define a webhook as a callback (executed/triggered) or a message (sent)

Comment (vykhrystiuk) [reply openapi/openapi.yaml:107]: I thought about "triggered"

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -104,6 +104,7 @@ tags:
       If it is not, or if you the `credit-file-proof` isn't requested, then redirect your customer to the KYC document gatherer to collect their KYC documents.
       Rebilly parses and analyzes the documents and accepts or rejects them according to a configurable scoring algorithm.
       When all of the document types in the KYC request are accepted, the status is considered fulfilled and the KYC request fulfilled webhook will be emitted.
```
Comment (subenson) [inline openapi/openapi.yaml:106]:
```suggestion
      When all of the document types in the KYC request are accepted, the status is considered fulfilled and the KYC request fulfilled webhook will be triggered.
```

File: openapi/openapi.yaml [hunk 4]
```diff
@@ -104,6 +104,7 @@ tags:
...
+      If a document is accepted while unreviewed documents of the same type exist, the unreviewed documents will be archived and the archived webhook will be emitted.
```
Comment (subenson) [inline openapi/openapi.yaml:107]:
```suggestion
      If a document is accepted while unreviewed documents of the same type exist, the unreviewed documents will be archived and the archived webhook will be triggered.
```

**Comment by subenson** [comment]
@Rebilly/php @Rebilly/js @Rebilly/docs please review

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**